selene/core/entity.py

```diff
@@ -174,26 +176,31 @@ def __call__(self) -> WebElement:
     # --- WaitingEntity --- #
 
     @property
-    def wait(self) -> Wait[E]:  # todo:  will not it break code like browser.with_(timeout=...)?
+    def wait(self) -> Wait[E]:
+        # todo:  will not it break code like browser.with_(timeout=...)?
```

Comment:
I moved todo on the new line manually in case not to let black format the code like 
```python
def wait(
    self,
) -> Wait[E]:  # todo:  will not it break code like browser.with_(timeout=...)?
```